test/utils/urlMatcher.test.ts

```diff
@@ -41,6 +41,55 @@ test('getTwitterUsername', () => {
     expect(getTwitterUsername('twitter.com/123/')).toBe('123')
 })
 
+test('isGithubPullRequestPage', () => {
+  expect(isGithubPullRequestPage('https://www.github.com/')).toBe(false)
+  expect(isGithubPullRequestPage('https://www.github.com/pull')).toBe(false)
+  expect(isGithubPullRequestPage('https://www.github.com/pull/')).toBe(false)
+  expect(isGithubPullRequestPage('https://www.github.com/username/repo/pull/123')).toBe(true)
+  expect(isGithubPullRequestPage('https://www.github.com/username/repo/pull/123/')).toBe(true)
+  expect(isGithubPullRequestPage('www.github.com/username/repo/pull/123')).toBe(true)
+  expect(isGithubPullRequestPage('github.com/username/repo/pull/123/')).toBe(true)
+  expect(isGithubPullRequestPage('abcd')).toBe(false)
```

Comment:
```suggestion
  expect(isGithubPullRequestPage('https://google.com/')).toBe(false)
```
This would be a better test